DentOS_Framework/DentOsTestbedLib/src/dent_os_testbed/lib/bridge/linux/linux_bridge_vlan_impl.py

```diff
@@ -22,8 +22,8 @@ def format_update(self, command, *argv, **kwarg):
             cmd += "dev {} ".format((params["device"]))
         if "vid" in params:
             cmd += "vid {} ".format((params["vid"]))
-        if "pvid" in params:
-            cmd += "pvid {} ".format((params["pvid"]))
+        if "pvid" in params and params["pvid"]:
```

Comment:
Can be simplified to if params.get("pvid", None) - since you are checking if the key is in dict and the value is not empty, get() will return you either an empty value or None, and both of them will make this condition False.